src/ui/char_selector.py

```diff
@@ -6,26 +6,27 @@
 
 
 class CharSelector:
+    _last_char_template = None
```

Comment:
This was his solution to a prior discussion we had about this PR breaking character recovery after restart since moving the logic from game controller to bot but then creating a new instance of bot on game restart.

Reply:
It's a bit nasty to store it like that but we need to store it among Bot instances. I can create a separate singleton Storage object but I think  it's reasonably encapsulated and should fit the purpose for the time being